.github/workflows/main.yml

```diff
@@ -92,18 +90,15 @@ jobs:
       if: matrix.type == 'cython'
       env:
         CYTHON_IGNORE: True
-        DISABLE_SENTRY: True
       run: |
         cd OctoBot
-        pytest --cov=. --cov-config=.coveragerc --durations=0 -rw tentacles
+        pytest --cov=. --cov-config=.coveragerc --durations=0 -rw tentacles/Backtesting tentacles/Evaluator tentacles/Services tentacles/Trading/Mode
```

Comment:
should we use `--ignore` https://docs.pytest.org/en/6.2.x/example/pythoncollection.html ?

Reply:
Right, that's better 👍